### DISCUSSION TITLE
No aircraft in moving map FBW A320neo leap?

### DISCUSSION THREAD

**Discussion by Noboru00** [flybywiresim/aircraft#4883] [Help] [answered]
I had posted this question previously but had not seen it posted or answered why?
 Used the FlyByWire installer to install version 6.1 on win 64x 10 updated, MSFS 2020 store version.
Please explain possible issues that might prevent the aircraft from not being displayed on moving map.
The simbrief flightplan comes in fine and all systems are go.

Thank You
Noboru

**Comment by hotshotp** [comment, marked as answer]
telex being off in options will prevent you from being seen on the map, when you got it on, you will be able to be shown on the map

**Reply by Noboru00** [reply]
What is telex? I looked in FMC in option and the EFB for it. Would that be messaging?

Thank You

**Reply by hotshotp** [reply]
> What is telex? I looked in FMC in option and the EFB for it. Would that be messaging?
> 
> Thank You
> 

Mcdu menu -> options -> atsu (I think) -> free text (I think it's called either free text or telex)
